### DISCUSSION TITLE
Chaff Harvesting

### DISCUSSION THREAD

**Discussion by galon007** [Courseplay/courseplay#7094] [Q&A] [answered]
Hello guys.
Its been a while since i played FS19 last time but im back and updated all my old mods Courseplay included.
I just wondering if Chaff harvesting was removed from the mod. I can not set my tractor or harvester to follow it on left or right side anymore. If i remember there was option to do this and it worked really well.
Now my tractors following harvester from behind and it is not very stable way and most likely tractor got stuck in traffic on next turn.
Thank you

**Comment by Tensuko** [comment, marked as answer]
This may help you:
https://github.com/Courseplay/courseplay/wiki/Mode-2-and-Mode-3-FAQ
And Video how it works:
https://youtu.be/sT3fKcI6wqw

Important to know is:
Chopper with short Pipe works better, due to how GIANTS let the Chopper throw the chaff.
When on Gras Field we always drive behind chopper, becouse the Gras when chopped is reported as Fruit.
